cmdstanpy/stanfit.py

```diff
@@ -1116,6 +1115,28 @@ def optimized_params_dict(self) -> OrderedDict:
         """Returns optimized params as Dict."""
         return OrderedDict(zip(self.column_names, self._mle))
 
+    def stan_variable(self, name: str) -> np.ndarray:
```

Comment:
Similar to the existing `stan_variable` in the MCMC object, these new methods will want fairly detailed docstrings, describing at least the dimensionality of the return type.